slave/process-scim/bin/process-scim.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# Slave script for SCIM service. It gets data sent by Perun system and optionally run external script for process SCIM files.
+#
+#author:  Michal Prochazka
+#date:	  2017-08-01
+#
+#
+
+PROTOCOL_VERSION='1.0.0'
+
+function process {
+	DST_DIR="/var/spool/perun/"
```

Comment:
I don't think you need this directory. You can send them to the executed script as parameters.

Reply:
I want to use diff_mv, so I need to have a local copy.

Reply:
In this case this is ok.
But using diff_mv this way you have different problem: "What if script makes diff_mv with not-null return code and then ends with error? Another run of script will not change anything, because diff_mv will be without any change."

Reply:
If there is an error inside diff_mv, then it raises log_msg with error message, so it ends with exit non 0 value.